packages/web-app-files/src/spaces/views/Spaces.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="oc-p-s">
+    <h2 class="" v-text="$gettext('Spaces')"></h2>
```

Comment:
Maybe remove the empty classes if not needed? Tag could also be self-closing (just nitpicking here)